### ISSUE TITLE
Add custom modules?

### ISSUE THREAD

**Issue by ssteinerx** [jreese/markdown-pp#42]
Over in #40 you said:

> @jreese 
> then I'd rather see those implemented as custom modules instead.

Is there a way to add modules that I just missed?  As far as I can tell, whatever's in the `MarkdownPP/Modules` directory gets pulled in but I don't see where the code looks elsewhere.

Even a command line switch like:

    -m <extra-modules-path>

Possibly repeated and/or with paths delimited by e.g. ":" would probably be sufficient.

Looks like it'd just have to be searched and added to the list after the built-in modules by exactly the same mechanism and, assuming they conform to the very simple API, should work a treat as they used to say in the olden days.

Yes, no, maybe?

**Comment by jreese** [comment]
Yeah, adding support for pulling from either a standard path (`.markdownpp` maybe?) or from a path specified as a flag would be a good start.  Happy to review any pull requests, of course. :)

**Comment by neosformpeople** [comment]
9
